common/lib/xmodule/xmodule/js/src/video/07_video_volume_control.js

```diff
@@ -39,12 +39,24 @@ function() {
 
         template: [
             '<div class="volume">',
-                '<a href="#" role="button" aria-disabled="false" title="',
-                    gettext('Volume'), '" aria-label="',
-                    gettext('Click on this button to mute or unmute this video or press UP or DOWN buttons to increase or decrease volume level.'),
-                    '"></a>',
-                '<div role="presentation" class="volume-slider-container">',
-                    '<div class="volume-slider"></div>',
+                '<button class="control" aria-disabled="false" aria-label="',
+                    gettext('Volume: Click on this button to mute or unmute this video or press UP or ',
```

Comment:
@andy-armstrong Hmm, when this was on one line I was getting quality failures saying the line was too long. Thoughts?